src/stores/CheckoutController.ts

```diff
@@ -245,3 +256,84 @@ export const paypalPayment = async (successUrl: string, cancelUrl: string, error
     });
   });
 };
+
+export const updatePaypalPaymentMethod = async (
```

Comment:
Minor, but the rest of the code uses Pay**P**al

```suggestion
export const updatePayPalPaymentMethod = async (
```